### ISSUE TITLE
bug report: response error 400

### ISSUE THREAD

**Issue by hanbyul-kim** [glowingjade/obsidian-smart-composer#155]
Hi,

When I chat with smart composer, there're error responses occasionally.
The error message says all messages must have non-empty content, but I just used the chat same as usual.

```
plugin:smart-composer:294 Failed to generate response Error: 400 {"type":"error","error":{"type":"invalid_request_error","message":"messages.13: all messages must have non-empty content except for the optional final assistant message"}}
    at Me.generate (plugin:smart-composer:27:14174)
    at Ze.makeStatusError (plugin:smart-composer:27:6347)
    at Ze.makeRequest (plugin:smart-composer:27:7258)
    at async Lb.streamResponse (plugin:smart-composer:41:2531)
    at async nL.streamResponse (plugin:smart-composer:70:6783)
    at async eval (plugin:smart-composer:70:9022)
    at async Object.mutationFn (plugin:smart-composer:294:4189)
```

**Comment by kevin-on** [comment]
Hi, this error occurs when there are consecutive user messages, resulting in an empty assistant message. This is resolved by #156 . Thanks for reporting the issue.

**Comment by hanbyul-kim** [comment]
Thank you. I'm looking forward to the update!
